src/custom-blocks/banner/style.scss

```diff
@@ -19,6 +19,13 @@ Block: Banner (Frontend styles)
 			margin-bottom: 0;
 			line-height: 100%;
 		}
+
+		h1, h2, h3, h4, h5, h6 {
+			@include govuk-media-query($from: tablet) {
```

Comment:
Is this specific to this block or could it apply to all our blocks?

Reply:
specific to this block